<?xml version="1.0"?>
<launch>
  <arg name="monitor_namespace"/>
  <arg name="monitored_namespace" default=""/>
  <arg name="robot_description" default="robot_description"/>
  <arg name="discrete_plugin" default="BulletDiscreteBVHManager"/>
  <arg name="continuous_plugin" default="BulletCastBVHManager"/>
  <arg name="joint_state_topic" default=""/>
  <arg name="publish_environment" default="false"/>

  <node pkg="tesseract_sensor_simulator" type="tesseract_sensor_simulator_node" name="tesseract_sensor_simulator_node" output="screen">
    <param name="robot_description" type="string" value="$(arg robot_description)"/>
    <param name="discrete_plugin" type="string" value="$(arg discrete_plugin)"/>
    <param name="continuous_plugin" type="string" value="$(arg continuous_plugin)"/>
    <param name="joint_state_topic" type="string" value="$(arg joint_state_topic)"/>
    <param name="monitor_namespace" type="string" value="$(arg monitor_namespace)"/>
    <param name="monitored_namespace" type="string" value="$(arg monitored_namespace)"/>
    <param name="publish_environment" type="bool" value="$(arg publish_environment)"/>
  </node>

</launch>
